### ISSUE TITLE
Highlighted object on stack view should be shown on the overview panel

### ISSUE THREAD

**Issue by kylepollina** [kylepollina/objexplore#16]
When I have the stack view open, I would like the preview layout on the right side to update with the dir/preview/type/docstring of the highlighted object on the stack.

Currently, the explorer and preview windows only explore the object last on the stack.

![Screen Shot 2021-10-09 at 10 00 20 AM](https://user-images.githubusercontent.com/13981456/136667638-3caf9b5d-2ab6-4ce1-9c3c-1f847fb29e8a.png)

Basically I want everything to update based on what's currently highlighted on the stack.
